lib/daru/index/index.rb

```diff
@@ -222,6 +222,31 @@ def reorder(new_order)
       self.class.new(new_order.map { |i| from[i] })
     end
 
+    # Sorts a `Index`, according to its values. Defaults to ascending order sorting.
+    #
+    # == Options:
+    #
+    # ascending::
+    #   This can be `true` or `false`. False, to get descending order.
+    #
+    # == Returns:
+    # sorted `Index` according to its values.
+    #
+    # == Usage
+    #
+    #   di = Daru::Index.new [100, 99, 101, 1, 2]
+    #   # Say you want to sort descending order
+    #   di.sort(ascending: false)
+    def sort opts={}
+      opts = {ascending: true}.merge(opts)
+      if opts[:ascending]
+        new_index, = @relation_hash.sort.transpose
+      else
+        new_index, = @relation_hash.sort.reverse.transpose
+      end
```

Comment:
Yep, multi-line ternary operator is unacceptable. 
But!
You can have it in one line (if line length is not too large), or, alternatively, have 
```ruby
new_index = if ..... 
```
To show that entire `if` is about calculating this variable.